__tests__/test.test.ts

```diff
@@ -0,0 +1,5 @@
+import { test } from '@jest/globals';
+
+describe('trivial test', () => {
+    test('trivial test', async () => true);
+});
```

Comment:
You can just remove `__tests__` if there's no actual tests

Reply:
In that case `yarn test` will return 1. That probably requires some changes to the GH tasks?

Reply:
Remove this line: https://github.com/three-consulting/epoc-frontend/blob/refactor/major-refactor/.github/workflows/housekeeping.yml#L29